airflow/settings.py

```diff
@@ -251,16 +251,24 @@ def prepare_classpath():
         sys.path.append(PLUGINS_FOLDER)
 
 
-try:
-    from airflow_local_settings import *  # noqa F403 F401
-    log.info("Loaded airflow_local_settings.")
-except Exception:
-    pass
+def import_local_settings():
+    import airflow_local_settings
+
+    for k, v in airflow_local_settings.__dict__.items():
+        if not k.startswith("__") and not k.endswith("__"):
+            globals()[k] = v
+    print("Loaded airflow_local_settings from " + airflow_local_settings.__file__ + ".")
 
 
 def initialize():
     configure_vars()
     prepare_classpath()
+
+    try:
+        import_local_settings()
+    except ImportError:
+        pass
```

Comment:
Could you move the try/except in to the new function you've added please?

Reply:
Moved and tests updated.